src/EPR.Payment.Service/Strategies/RegistrationFees/Producer/BaseFeeCalculationStrategy.cs

```diff
@@ -0,0 +1,32 @@
+﻿using EPR.Payment.Service.Common.Constants.RegistrationFees;
+using EPR.Payment.Service.Common.Data.Interfaces.Repositories.RegistrationFees;
+using EPR.Payment.Service.Common.Dtos.Request.RegistrationFees;
+using EPR.Payment.Service.Common.ValueObjects.RegistrationFees;
+using EPR.Payment.Service.Strategies.Interfaces.RegistrationFees;
+
+namespace EPR.Payment.Service.Strategies.RegistrationFees.Producer
+{
+    public class BaseFeeCalculationStrategy : IFeeCalculationStrategy<ProducerRegistrationFeesRequestDto>
+    {
+        private readonly IProducerFeesRepository _feesRepository;
+
+        public BaseFeeCalculationStrategy(IProducerFeesRepository feesRepository)
+        {
+            _feesRepository = feesRepository ?? throw new ArgumentNullException(nameof(feesRepository));
+        }
+
+        public async Task<decimal> CalculateFeeAsync(ProducerRegistrationFeesRequestDto request, CancellationToken cancellationToken)
+        {
+            // If ProducerType is empty, return a base fee of zero
```

Comment:
Should we return a 0 fees when the producertype is empty? This is the AC;

**Scenario: Error handling for invalid producer type**
Given a producer has submitted the required data for registration with a regulator
When the producer type input parameter is invalid
Then an error message must be returned by PayCal API stating <producer type is invalid and fees cannot be returned>

Reply:
As discussed, empty producertype is a valid scenario where the base fee is not required, and only the subsidiaries fee is needed.